TypeCobol.LanguageServer/TypeCobolLanguageServer.cs

```diff
@@ -184,13 +187,19 @@ public RefreshOutlineParams UpdateOutline(ProgramClassDocument programClassDocum
             {
                 if (_rootOutlineNode == null)
                 {
-                    _rootOutlineNode = new OutlineNode(programClassDocument.Root);
+                    _rootOutlineNode = OutlineNode.BuildFrom(programClassDocument.Root);
                 }
 
                 if (bForced || _rootOutlineNode.Update(programClassDocument.Root))
-                    return new RefreshOutlineParams(new TextDocumentIdentifier(this.LspTextDocument.uri),
-                        _rootOutlineNode);
+                {
+                    return new RefreshOutlineParams()
+                    {
+                        textDocument = new TextDocumentIdentifier() { uri = this.LspTextDocument.uri },
+                        outlineNodes = _rootOutlineNode.childNodes
```

Comment:
As I removed the constructor for `RefreshOutlineParams`, the code here does the same thing. This is because the params expect a list of nodes whereas the class works with a 'fake' root node.